routes/index.js

```diff
@@ -4,11 +4,18 @@ var Product = require('../models/product');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { products: /* ??? */ });
+  res.render('index', { products: Product.all });
 });
 
 router.post('/cart', function(req, res, next) {
   // How do we add an item to the cart?
+  var cookieObject = parseInt(req.body.id);
```

Comment:
This works, but the name of the variable is a bit misleading. `cookieObject` is actually just the id, so I'd probably rename that variable to something that makes more sense for what it is.